g2p/mappings/langs/font-encodings/myh_doulos.json

```diff
@@ -0,0 +1,282 @@
+[
+  {
+    "in": "\\ ",
```

Comment:
What are all these \\ rules about?

Reply:
Ah, that is an artifact of the rules being semi-automatically generated from the character map in the PDF.  I'll remove it!

Reply:
but also, we need backslashes because the 'in' is a regex

Reply:
(I really don't *like* that it's a regex because it means we can't generally express g2p rules as FSTs, but it is what it is - perhaps we should define a subset of Python regex language that we accept)